extension/lib/squire-raw.js

```diff
@@ -0,0 +1,4990 @@
+/* Copyright © 2011-2015 by Neil Jenkins. MIT Licensed. */
```

Comment:
Thanks. Eventually we'll probably lean more into using npm instead, which will make updates easier.

Reply:
Yup, that'd be a great!